### PR TITLE
[#2] Add basic brick interface with accounts checkboxes

### LINKED ISSUE #2: Basic interface with brick
Integrate [`brick`](https://hackage.haskell.org/package/brick) library.

As the start point, we can show the existing accounts.

### LINKED ISSUE #3: Account type
Haskell module for accounts `Treasure.Account`

### PR DESCRIPTION
**Pull request by vrom911**
Kinda resolves #2 
Resolves #3 

*Sad note:* `brick` has issues with Unicode symbols, probably we will decide to give up on them if the solution is not found 😞

### CODE REVIEW

File: app/Main.hs [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import TreasureKeeper (someFunc)
+import Treasure.Tui (executeTreasure)
 
 main :: IO ()
-main = someFunc
-
+main = executeTreasure
```
Comment (ChShersh) [inline app/Main.hs]: Is it possible to create funnier name?.. Like, `monitorTreasure` (this is not that funny as well). Just an idea...

File: src/Treasure/Tui/Chest.hs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+         TreasureChest (..)
+       , CheckBox (..)
+       , toCheckBoxes
+
+         -- * Lenses
+       , chestAccountsL
+       , checkBoxL
+       ) where
+
+import Control.Lens (Lens', ix, lens, (.~))
```
Comment (ChShersh) [inline src/Treasure/Tui/Chest.hs]: I noticed that `brick` depends on the `microlens` library. I was thinking earlier that `lens` can help us better but now I see that everything we need is in `microlens` and `microlens-th` packages. So we probably can depend on `microlens-*` libraries to make `treasure-keeper` slightly smaller and build faster

File: src/Treasure/Tui/Chest.hs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    } deriving (Show)
+
+-- | Creates the list of the 'CheckBox'es from the list of data with the 'False'
+-- flags.
+toCheckBoxes :: [a] -> [CheckBox a]
+toCheckBoxes = map (`CheckBox` False)
+
+-- | Lens for 'checkboxFlag' field of the 'CheckBox' data type.
+flag :: Lens' (CheckBox a) Bool
+flag = lens checkboxFlag $ \checkbox newFlag -> checkbox { checkboxFlag = newFlag }
```
Comment (ChShersh) [inline src/Treasure/Tui/Chest.hs]: Do you think this should be `flagL` to have the same convention for all lenses?

File: treasure-keeper.cabal [hunk 4]
```diff
@@ -25,11 +25,17 @@ library
 
   exposed-modules:     TreasureKeeper
```
Comment (ChShersh) [inline treasure-keeper.cabal]: Maybe we can remove `TreasureKeeper` module?

File: app/Main.hs (hunk 1 above)
Comment (vrom911) [reply app/Main.hs]: `openTreasureChest`?

File: treasure-keeper.cabal (hunk 4 above)
Comment (vrom911) [reply treasure-keeper.cabal]: 🤔 For now yes. But maybe it's better to have this `executeTreasure` function there?

Comment (ChShersh) [reply treasure-keeper.cabal]: Hmm, this is a good idea as well! Can can

**Pull request merged**